src/components/unique-token/Tag.js

```diff
@@ -16,17 +16,21 @@ const HairlineSpace = '\u200a';
 
 const PropertyActionsEnum = {
   openURL: 'openURL',
-  viewTraitOnOpensea: 'viewTraitOnOpensea',
+  viewTraitOnNftMarketplace: 'viewTraitOnNftMarketplace',
 };
 
-const viewTraitOnOpenseaAction = {
-  actionKey: PropertyActionsEnum.viewTraitOnOpensea,
-  actionTitle: lang.t('expanded_state.unique_expanded.view_all_with_property'),
-  discoverabilityTitle: 'OpenSea',
-  icon: {
-    iconType: 'SYSTEM',
-    iconValue: 'magnifyingglass.circle.fill',
-  },
+const getViewTraitOnNftMarketplaceAction = (marketplaceName = 'OpenSea') => {
```

Comment:
can now remove marketplaceName default of OpenSea as it is now included in parsers explicitly